### PR TITLE
Reselecting same entity in list will unselect it

### PR DESCRIPTION
**Pull request by dback2**
Fix the same entity from being re-selectable multiple times when holding Ctrl and clicking on it in the entity list. When re-selecting the same entity it will be considered unselecting it instead.

Also fix indentations in entityList.js.

Fixes: https://highfidelity.manuscript.com/f/cases/17294/

Test Plan:
- Enter Interface in desktop mode
- Open Create and select an entity in the entity list
- Verify the entity row gets highlighted, the correct entity is selected in world, and that "1 entity selected appears" at the bottom of the list
- Hold Ctrl key and select the same entity in the entity list again
- Verify the entity gets unselected both in the list and in world, and the bottom text goes back to "X entities found"
- Hold Ctrl key and select 2+ entities in the entity list
- Verify the correct entities get selected in world, upon selecting the 2nd entity the edit handles change to multi-selection (corner scale cubes and scale spheres are no longer present and bounding box encompasses all selected entities), both entity rows are highlighted in the list, and "X entities selected" text appears for number selected
- While holding Ctrl key re-select a selected entity in the list
- Verify that the correct entity gets unselected both in the list and in world, and the bottom text's "entities selected" counter goes down by 1

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11909/HighFidelity-Beta-PR13771-174519e.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11909/HighFidelity-Beta-PR13771-174519e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11909/HighFidelity-Beta-Interface-PR13771-174519e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11909/HighFidelity-Beta-PR13771-174519e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11909/HighFidelity-Beta-Interface-PR13771-174519e.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Paused Testing, awaiting dev response

**Comment by ctrlaltdavid** [comment]
If you Ctrl-click on an entity in the list that's already selected, it should be deselected (i.e., Ctrl-click should toggle the entity's selected/unselected state). This is a common way of doing things in Windows.

**Comment by dback2** [comment]
I agree, we were just talking about that earlier as it makes more sense, and should have the change soon.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11947/HighFidelity-Beta-PR13771-a04ffd9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11947/HighFidelity-Beta-PR13771-a04ffd9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11947/HighFidelity-Beta-Interface-PR13771-a04ffd9.dmg

**Comment by QliCknox** [comment]
Pass @conklin94122 

Test performs as expected and user is able to select multiple entities

File: scripts/system/html/js/entityList.js [hunk 1]
```diff
@@ -28,436 +28,442 @@ debugPrint = function (message) {
...
+        document.getElementById("entity-hasScript").onclick = function () {
+            setSortColumn('hasScript');
+        };
+
+        function onRowClicked(clickEvent) {
+            var id = this.dataset.entityId;
+            var selection = [this.dataset.entityId];
+            if (clickEvent.ctrlKey) {
+                var selectedIndex = selectedEntities.indexOf(id);
+                if (selectedIndex >= 0) {
```
Comment (dback2) [inline scripts/system/html/js/entityList.js]: The actual change is here, the rest is all fixing indentations.

**Pull request merged**